components/bpmn/bpmn-explorer-web/src/web/js/actions.js

```diff
@@ -1005,3 +1004,446 @@ function generateReport(){
+    getCalledProcessInstancesOfCompleted(id);
+
+}
+
+//Gets the details of all the activities in a running/active process instance
+function getAuditLogForRunningProcessInstances(pid,id){
+
+    var url = "/" + CONTEXT + "/send?req=/bpmn/stats/processTaskServices/allTasks/" + pid;
+
+    $.ajax({
+        type: 'GET',
+        contentType: "application/json",
+        url: httpUrl + url,
+        success:
+
+            function innerFunction(data){
+
+                var taskList = JSON.parse(data);
+                
+                var url1 = "/" + CONTEXT + "/send?req=/bpmn/history/historic-activity-instances?processInstanceId=" + id;
+
+                $.ajax({
+                    type: 'GET',
+                    contentType: "application/json",
+                    url: httpUrl + url1,
+                    success: function (data){
+
+                        $("#auditLog").html("");
+                        var taskList2 = JSON.parse(data);
+                        
+                        var DIV = "<div style='height:100%;overflow:auto;'><table id ='table1'><thead><td>State</td><td>Activity Name</td><td>Activity Type</td><td>Start Time</td><td>End Time</td><td>Task Id</td><td>Activity Instance Id</td></thead><tbody>"
+
+                        for (var k = 0; k < taskList.data.length; k++) {
+
+                            var activityName  = taskList.data[k].name;
+                            var taskDefKey  = taskList.data[k].taskDefinitionKey;
+                            var activityType  = taskList.data[k].type;
+
+                            for (var j = 0; j < taskList2.data.length; j++) {
+
+                                var activityId  = taskList2.data[j].activityId;
+                                var startTime = taskList2.data[j].startTime;
+                                var endTime = taskList2.data[j].endTime;
+                                var taskId = taskList2.data[j].taskId;
+                                var activityInstanceId = taskList2.data[j].id;
+                                if (taskId == null) {
+                                    taskId = "N/A";
+                                }
+                                if (taskDefKey == activityId && endTime !== null ) {
+                                    var state = "Completed";
+                                    break;
+                                } else if (taskDefKey == activityId && endTime == null ) {
+                                    var state = "Active";
+                                    var endTime = "N/A";
+                                    break;
+                                } else {
+                                    var state = "Not Started";
+                                    var startTime = "N/A";
+                                    var endTime = "N/A";
+                                    var activityInstanceId = "N/A";
+                                    var taskId = "N/A";
```

Comment:
No need to declare variables here again since declared in line 1209-1213. Just set values